distribution/docs/pom.xml

```diff
@@ -24,6 +24,8 @@
     <properties>
         <skipTests>true</skipTests>
         <ddf.version>${project.version}</ddf.version>
+        <timestamp>${maven.build.timestamp}</timestamp>
```

Comment:
For the pdf License page. There's a `Last updated...` line that was only working in html.